source/jormungandr/jormungandr/realtime_schedule/sytral.py

```diff
@@ -115,17 +115,19 @@ def _make_url(self, route_point):
         The url returns something like a departure on a stop point
         """
 
-        stop_id = route_point.fetch_stop_id(self.object_id_tag)
+        stop_id_list = route_point.fetch_all_stop_id(self.object_id_tag)
 
-        if not stop_id:
+        if not stop_id_list:
             logging.getLogger(__name__).debug(
-                'missing realtime id for {obj}: stop code={s}'.format(obj=route_point, s=stop_id),
+                'missing realtime id for {obj}: stop code={s}'.format(obj=route_point, s=stop_id_list),
                 extra={'rt_system_id': unicode(self.rt_system_id)},
             )
             self.record_internal_failure('missing id')
             return None
 
-        url = "{base_url}?stop_id={stop_id}".format(base_url=self.service_url, stop_id=stop_id)
+        url = self.service_url + "?stop_id={stop_id}".format(stop_id=stop_id_list[0])
+        for stop_id in stop_id_list[1:]:
+            url += "&stop_id={stop_id}".format(stop_id=stop_id)
```

Comment:
concataining string isn't very efficient

Reply:
I'm listening all proposition that is more pythonic and faster

Reply:
If I understood correclty, you thought about to use **request.get** with the _request python lib_,
but there is a **circuit breaker** [here](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/realtime_schedule/sytral.py#L94).

I have seen that inside all navitia code we use it with a _string url param_. So, we need to concanate before the _breaker.call_

Am I right ?

Reply:
As I said in Meeting, I prefer to reshape the url part in an other PR for clarity and to release fastly the feature inside this PR.